File: 11 - Transform Array by Multiplying Even Indices and Subtracting Odd Indices/solution.js
function transformArray(arr) {
  // Your code here
  let n = arr.length;

  for (let i = 0; i < n; i++) {
    if (i % 2 === 0) {
      [arr[i]] = [arr[i] * i];
    } else {
      [arr[i]] = [arr[i] - i];
    }
  }

  return arr;
}

// Test Cases
console.log(transformArray([4, 3, 6, 2, 5]));

File: 11 - Transform Array by Multiplying Even Indices and Subtracting Odd Indices/solution.py
def transform_array(arr):
    # Your code here
    n = len(arr)

    for i in range(n):
        if i % 2 == 0:
            arr[i] *= i
        else:
            arr[i] -= i

    return arr

# Test Cases
print(transform_array([4, 3, 6, 2, 5]))